org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/services/XMLRenameTest.java

```diff
@@ -157,6 +157,24 @@ public void testNamespaceRenameStartTagSuffix() throws BadLocationException {
 			assertRename(xml, "BB", edits("BB", r(2, 6, 7), r(2, 13, 14))); 
 	}
 
+	@Test
+	public void testRenameComplexTypeName() throws BadLocationException {

```

Comment:
This test must be remove in the XSD extension in XSDRenameExtensionTest (like XSDDefinitionExtensionTest)